solutii/andreea.rosca/chess/chess.css

```diff
@@ -0,0 +1,104 @@
+body {
```

Comment:
Nu trebuie sa adaugi un fisier in acest director. Trebuie respectata structura prezentat la laborator.
Directorul corect este `web/solutii/nume_prenume/*`
Pentru mai multe informatii acceseaza [pagina cursului](https://teaching.alexcoman.com/resurse/tutorial/git/2017/03/11/git-propunerea-unei-solutii/).